### DISCUSSION TITLE
Is this production ready / used already by companies in production?

### DISCUSSION THREAD

**Discussion by mikevercoelen** [kartikk221/hyper-express#219] [Q&A] [answered]
Title says it all, I'm curious.

**Comment by kartikk221** [comment, marked as answer]
Hey, I have personally been using HyperExpress in production for over 2 years now in my own SaaS products. I also know a bunch of startups personally using HyperExpress as well. Now this is nowhere as big as Fastify or Express but I am actively maintaining it as I am using it production myself, so I would assume to an extent, It would be production ready.

**Reply by mikevercoelen** [reply]
Got it, thanks for the quick reply!
